Cython/Compiler/Optimize.py

```diff
@@ -2370,12 +2370,35 @@ def _handle_method(self, node, type_name, attr_name, function,
         can additionally resolve bound and unbound methods here that were
         assigned to variables ahead of time.
         """
+
         if kwargs:
             return node
-        if not function or not function.is_attribute or not function.obj.is_name:
+        if not function or not function.is_attribute:
+            return node
+
+        if hasattr(node, "args"):
+            args = node.args
+            if args is None and node.arg_tuple:
+                args = node.arg_tuple.args
+    
+            if attr_name == "bit_count":
```

Comment:
Again, this should not happen here. Please move it into the handler method that you added further down.